spec.md

```diff
@@ -990,6 +990,9 @@ sub-element in the vector.
 Invalidating a component with a bundle type recursively invalidates each
 sub-element in the bundle.
 
+Invalidating a particular subset of bits in an integer is possible by
```

Comment:
This is starting to sound like invalid is being used for don't care.
For normal wires/regs/outputs, how does invalidating a subset of bits get you anything that starting with an entire invalid value which is later partially written not get you?